### ISSUE TITLE
ModelDoc/Animgraph - Bone merge cloud models

### ISSUE THREAD

**Issue by KaesarZZ** [Facepunch/sbox-issues#4900]
### For?

S&Box

### What can't you do?

I would like to use the bonemerge tool with cloud models since it looks like they will all be in asset.party.

Example the official weapons, the option is there but the model appears at point 0 even if you have the playermodel prepared with the hold_R. (You can test with the citizen)

https://i.gyazo.com/dc973fbcf99bdc358400926627f501d8.mp4

### How would you like it to work?

I would like it to be merged correctly.

### What have you tried?

Right now you need to download the model from the cloud, then look it up manually and put it in your addon and that sucks.

**Comment by garrynewman** [comment]
I don't think there's any reason this wouldn't be possible, the cloud model is just a model.

**Comment by aylaylay** [comment]
I don't think this has anything to do with cloud models, the bone merge code they use in modeldoc just doesn't handle this situation, I'll see if I can make it work

**Comment by aylaylay** [comment]
mp5 doesn't bone merge with citizen in game either?

**Comment by MrBrax** [comment]
<img width="340" alt="1710967424028" src="https://github.com/Facepunch/sbox-issues/assets/1517911/9f1f6b20-56e8-4edc-9530-0ab672b39fda">

ingame works for me but looks like total arse

**Comment by aylaylay** [comment]
How is that set up? Bone merge is for clothing, not weapons. If you want to attach a weapon, parent it to the bone.

**Comment by MrBrax** [comment]
gotcha, i did test both but i now have it set parented to the bone hold_R, and with a pistol with the pistol hold type, it's still off. is it just because of the absurdly large hands on the citizen?

<img width="407" alt="1710967742154" src="https://github.com/Facepunch/sbox-issues/assets/1517911/26f52f3b-b3fa-46dc-867f-2ebdff76259c">

**Comment by KaesarZZ** [comment]
Don't you use MP5 in sbox-arena? or at least I swear it worked in Sanbox pre-scene.
